src/main/default/lwc/voteCount/voteCount.js

```diff
@@ -0,0 +1,68 @@
+import { LightningElement, api } from 'lwc';
+import helpedUsers from '@salesforce/label/c.cookbook_HelpedUsers';
+import helpedUsers_plural from '@salesforce/label/c.cookbook_HelpedUsers_plural';
+import helpedUsers_zero from '@salesforce/label/c.cookbook_HelpedUsers_zero';
+import { I18nUtils } from 'c/quanticUtils';
+
+/**
+ * The `voteCount` component is an indicator of how many people said that a document was helpful.
+ * @example
+ * <c-vote-count active="true" count="102"></c-vote-count>
+ */
+export default class VoteCount extends LightningElement {
+  labels = {
+    helpedUsers,
+    helpedUsers_plural,
+    helpedUsers_zero
+  };
+
+  /**
+   * The count to be shown to the user.
+   * @type {number}
+   * @defaultValue `0`
+   */
+  @api count = 0;
```

Comment:
I'm wondering whether the component should be hidden when the vote count is zero.

Reply:
It can be yeah, I liked the idea of showing the message :  "Be the first to rate this document" when the count is zero to give the user a bit of a motivation to vote.

Reply:
:+1: I like that: "Be the first to rate this document"